### DISCUSSION TITLE
/getting-started/installation.html

### DISCUSSION THREAD

**Discussion by giscus** [goravel/goravel#121] [Announcements]
# /getting-started/installation.html

[[toc]] 服务器要求 Golang >= 1.18; 安装 Goravel 如果安装依赖较慢，请使用国内代理，详见文章 (https://learnku.com/go/wikis/38122)。 启动 HTTP 服务 指定 .env 文件启动服务 热更新 安装 cosmtrek/air (https://github.com/cosmtrek/a...

https://www.goravel.dev/zh/getting-started/installation.html

**Comment by nla-asia** [comment]
I guess there is a little typo here.. "go run. artisan key:generate"  >> "go run  .  artisan key:generate"

**Comment by nashihamm** [comment]
wow

**Reply by nashihamm** [reply]
why?

**Comment by nikultaka** [comment]
is it create schema migration same as laravel ? so far for golang migration is biggest headache

**Reply by hwbrzzl** [reply]
Yes, you can try it first.

**Comment by ifqygazhar** [comment]
Can i change the http using fiber?

**Reply by hwbrzzl** [reply]
Yes, please check https://github.com/goravel/fiber

**Reply by ifqygazhar** [reply]
wow that was amazing, thank you !!

**Reply by ifqygazhar** [reply]
ah ya i have another question, it's goravel can be combined with popular frontend library/framework like react js,vue,svelte ?, like on laravel they have inertia.js

**Reply by hwbrzzl** [reply]
Goravel doesn't adapt intertia.js yet, but there is a simple example to show how to integrate a single page app in Goravel: https://github.com/goravel/example/blob/v1.16.x/routes/web.go#L24-L29

**Reply by ifqygazhar** [reply]
okay thank youu !!

**Comment by altalya** [comment]
If anyone faces goravel command not found issue, follow the steps to add the path in linux
```
ls ~/go/bin #To find goravel is there
nano ~/.bashrc
export PATH=$PATH:$HOME/go/bin
source ~/.bashrc
goravel --version


```
